### PR TITLE
Selenium test example

### PR DESCRIPTION
**Pull request by dimasciput**

### CODE REVIEW

**Comment by timlinux** [comment]
Ah that is cool @dimasciput ! Looks like we will be getting @Christiaanvdm to write lots of UI tests soon! Any issues with it running cleanly on travis?

File: deployment/production/REQUIREMENTS.txt [hunk 1]
```diff
@@ -61,3 +61,4 @@ django-crispy-forms==1.7.2
 djangorestframework-gis==0.13
 pandas==0.23.4
 git+https://github.com/dimasciput/pandas_access.git
+selenium==3.12.0
```
Comment (lucernae) [inline deployment/production/REQUIREMENTS.txt]: I think you don't need to install selenium on production mode?

File: deployment/ansible/development/group_vars/all.travis.yml [hunk 2]
```diff
@@ -29,6 +29,8 @@ django:
     DATABASE_HOST: db
     SITEURL: http://uwsgi/
     SECRET_KEY: 'tT\xd7\xb06\xf7\x9b\xff\x0fZL\xca\xca\x11\xefM\xacr\xfb\xdf\xca\x9b'
+    ON_TRAVIS: True
+    SELENIUM_DRIVER: 'http://hub:4444/wd/hub'
```
Comment (lucernae) [inline deployment/ansible/development/group_vars/all.travis.yml]: For travis, you need to replace this travis ip address: `http://172.17.0.1:4444/wd/hub`

File: deployment/docker-compose.travis.yml [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+      - DATABASE_HOST=db
+      - DJANGO_SETTINGS_MODULE=core.settings.prod_docker
+      - VIRTUAL_HOST=bims.kartoza.com
+      - VIRTUAL_PORT=8080
+      - RABBITMQ_HOST=rabbitmq
+      - HAYSTACK_HOST=elasticsearch
+      - GEOCONTEXT_URL=https://geocontext.kartoza.com
+      - GEOCONTEXT_COLLECTION_KEY=climate
+      - CONTACT_US_EMAIL=dimas@kartoza.com
+      - APP_NAME=bims
```
Comment (lucernae) [inline deployment/docker-compose.travis.yml]: Override `SELENIUM_DRIVER` environment variable here.

**Pull request merged**